doc/man7/property.pod

```diff
@@ -118,7 +118,7 @@ Ordering of optional clauses is not significant.
 
 In order to permit a more concise expression of boolean properties, there
 is one short cut: a property name alone (e.g. "default") is
-exactly equivalent to "default=yes" in both definitions and queries.
+exactly equivalent to "provider=default" in both definitions and queries.
```

Comment:
I'm quite uncomfortable with this short cut.  Booleans by default seems okay, providers a lot less so.

It is also not implemented :)

Reply:
This is actually more a case of me not properly reading the text I was updating rather than an intention to implement this short cut!

Reply:
Fixed.